src/dialog/overflow-menu/overflow-menu-option.component.ts

```diff
@@ -32,8 +33,26 @@ import {
 			(click)="onClick($event)"
 			[disabled]="disabled"
 			[attr.title]="title">
-			<ng-content></ng-content>
+			<ng-container *ngTemplateOutlet="tempOutlet"></ng-container>
```

Comment:
Why not `ng-content`?

Reply:
Couldn't have two ng-content in the template

Reply:
Is there a specific error? Cause I think you can ... You should be able to, given that we're `ngIf`ing out a whole `ng-content`

If it's a compiler issue then this is fine :+1:

Reply:
There's no error but the basic overflow completely breaks and you can't see any of the options

Reply:
Gotcha, cool we can leave this then :+1: